packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -123,6 +128,70 @@ class EmbeddedContentConsumer extends React.Component {
     this.addEventListeners(this.props.eventHandlers);
   }
 
+  handleFrameVisualFocusIndicator() {
+    // Selectors for interactable elements
+    const interactableElementSelector = 'a[href]:not([tabindex=\'-1\']), area[href]:not([tabindex=\'-1\']), input:not([disabled]):not([tabindex=\'-1\']), '
+      + "select:not([disabled]):not([tabindex='-1']), textarea:not([disabled]):not([tabindex='-1']), button:not([disabled]):not([tabindex='-1']), "
+      + "[contentEditable=true]:not([tabindex='-1'])";
+
+    // Check if the content is scrollable
+    const isContentScrollable = () => {
+      const doc = this.xfcFrame?.iframe?.contentWindow?.document;
+      return (doc.documentElement.scrollHeight > doc.documentElement.clientHeight
+        || doc.body.scrollHeight > doc.body.clientHeight
+        || doc.documentElement.scrollWidth > doc.documentElement.clientWidth
+        || doc.body.scrollWidth > doc.body.clientWidth);
+    };
+
+    // Check if the iframe has `scrolling` attribute set or not
+    // The default `scrolling` attribute is `auto`.
+    const scrollingEnabled = () => (this.xfcFrame?.iframe?.getAttribute('scrolling') !== 'no');
+
+    // Event listener and callback function for when `load` is completed for the content in the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('load', () => {
+      this.hasInteractableElement = [...this.xfcFrame?.iframe?.contentWindow?.document.body.querySelectorAll(`${interactableElementSelector}`)].some(
+        (element) => !element.hasAttribute('disabled')
+          && !element.getAttribute('aria-hidden')
+          && !!(element.offsetWidth || element.offsetHeight || element.getClientRects().length)
+          && window.getComputedStyle(element).visibility !== 'hidden'
+          && element.closest('[inert]') === null,
+      );
+
+      if (scrollingEnabled() && isContentScrollable() && this.hasInteractableElement === false) {
+        // Set tabIndex="0" so focus can go into the document when
+        // using tab key when scrolling is enabled
+        this.xfcFrame.iframe.contentWindow.document.body.tabIndex = 0;
+      }
+    });
+
+    // Event listener and callback function for `resize` event of the iframe
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('resize', () => {
```

Comment:
We should store the content window reference in a local variable.  It is used quite a bit in the file and would make the logic look cleaner.  We also would not have to look up the reference every time.